README.md

```diff
@@ -54,24 +54,24 @@ S2ORC is now available through the [Semantic Scholar Public API](https://www.sem
 
 ## Download instructions
 
-Please request access to S2ORC by:
-1. Requesting a Semantic Scholar API key [here](https://www.semanticscholar.org/product/api#Partner-Form)
-2.  **It may take us up to a week to get back to you.** If it has been longer than one week since you have completed the form and you have not heard from us, please send us an email -- your request may have slipped through the cracks.
-
+The original S2ORC dataset files were refactored into multiple datasets available through [the Semantic Scholar APIs](https://api.semanticscholar.org/) (See detailed documentation [here](https://api.semanticscholar.org/api-docs/datasets)). 
+For questions, feature requests, bug reports, please search existing issues on [the s2-folks Github repo](https://github.com/allenai/s2-folks/issues?q=is%3Aissue) before creating [a new issue](https://github.com/allenai/s2-folks/issues/new). 
 
 ## Contact us
 
 The best way to contact us is through email.  Don't hesitate to reach out about anything; we've helped a lot of people get started with the dataset, which can be a bit daunting given its size.
 
-**Email:** Please include `{kylel, lucyw, rodneyk, waleeda}@allenai.org` on all correspondence.
+**Email:** Please include `{kylel, lucyw, rodneyk}@allenai.org` on all correspondence.
 
 **Twitter** [@kylelostat](https://twitter.com/kylelostat), [@lucyluwang](https://twitter.com/lucyluwang)
 
 **IRC:** `#s2orc` at `irc.oftc.net`
 
 **Give us Feedback:**  Totally optional, but we'd love to hear how you're using this dataset & any feedback for improving it.  Send us an email or leave a Github Issue. 
 
-**Report issues:** Use [GitHub Issues](https://github.com/allenai/s2orc/issues) to report bugs or issues!  We'll try to fix it for the next release.
+**Report issues:** 
+* For questions, feature requests, bug reports, please search existing issues on [the s2-folks Github repo](https://github.com/allenai/s2-folks/issues?q=is%3Aissue) before creating [a new issue](https://github.com/allenai/s2-folks/issues/new). 
+* Use [GitHub Issues](https://github.com/allenai/s2orc/issues) to report bugs or issues in the original S2ORC dataset!  We'll try to fix it for the next release.
```

Comment:
It's unclear to me what kind of bugs/issues you'd like people to report in https://github.com/allenai/s2orc/issues. @kyleclo @lucylw

Reply:
We're not doing future releases, so maybe let's remove this last line